### ISSUE TITLE
Add ability to add alt text retroactively

### ISSUE THREAD

**Issue by alimony** [bluesky-social/social-app#659]
**Is your feature request related to a problem? Please describe.**

Once posted, you cannot add alt text to pictures or edit the existing one.

**Describe the solution you'd like**

It would be nice if alt text could be added or edited retroactively.

Original idea here: https://staging.bsky.app/profile/ruben.codes/post/3jvsx5anwad26

**Comment by Wilker-uwu** [comment]
i would appreciate a feature where others can contribute to the alt text in a way that the creator of the original post can approve, in posts where alt text is often missing.

this feature is specially helpful for people who posts a ton of images but none of which with alt text. in those cases i assume it would help more people start writing those texts on their own when posting new images

**Comment by scottdotweb** [comment]
> i would appreciate a feature where others can contribute to the alt text in a way that the creator of the original post can approve, in posts where alt text is often missing.

I was thinking of opening an issue specifically for this - the poster could be presented with "accept suggestion" and "edit suggestion" buttons. But this feature would need to be designed carefully so as not to be a potential vector for invisible abuse.

**Comment by cooljeanius** [comment]
> > i would appreciate a feature where others can contribute to the alt text in a way that the creator of the original post can approve, in posts where alt text is often missing.
> 
> I was thinking of opening an issue specifically for this - the poster could be presented with "accept suggestion" and "edit suggestion" buttons. But someone's beaten me to it with [#6559](https://github.com/bluesky-social/social-app/issues/6559).

Yeah I think that something like this could really help boost alt-text adoption; I've seen it suggested several times on-app lately, too... (I don't have the links to the posts at the moment now, though)
